apps/pipeline/src/services/answer-server/create-paragraph-embeddings.integration.spec.ts

```
@@ -0,0 +1,138 @@
+      characterActions: [],
+    },
+    chapterCharacters: [],
+  };
+}
+
+describe("generateEmbeddings integration (real paragraph extraction)", () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
+
+  it.skip("uses real chapter paragraph extraction with mocked getBookData/getMtimeMs/readBookFile", async () => {
```

ESLint

[error] Disabled test

(vitest/no-disabled-tests)